src/core/function-core.js

```diff
@@ -1,12 +1,11 @@
 const process = require('process');
 const fetch = require('node-fetch');
 
-const URL_CREATE_CHAT = "https://us-central1-whappu-183808.cloudfunctions.net/addNewChatBetweenUsers";
-const URL_ADD_PUSHTOKEN = "https://us-central1-whappu-183808.cloudfunctions.net/addPushTokenForUserId";
+const BASE_URL = "https://us-central1-whappu-183808.cloudfunctions.net";
```

Comment:
I still recommend turning this into environment variable

Reply:
I will. Thanks.